packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -150,6 +152,27 @@ const MenuInspectorControls = ( props ) => {
 					>
 						{ __( 'Menu' ) }
 					</Heading>
+					<BlockControls group="inline">
+						<NavigationMenuSelector
+							currentMenuId={ currentMenuId }
+							onSelectClassicMenu={ onSelectClassicMenu }
+							onSelectNavigationMenu={ onSelectNavigationMenu }
+							onCreateNew={ onCreateNew }
+							createNavigationMenuIsSuccess={
+								createNavigationMenuIsSuccess
+							}
+							createNavigationMenuIsError={
+								createNavigationMenuIsError
+							}
+							actionLabel={ actionLabel }
+							isManageMenusButtonDisabled={
+								isManageMenusButtonDisabled
+							}
+							icon={ chevronDown }
+							text={ __( 'Change' ) }
```

Comment:
Change is too generic for accessibility even in the context of the navigation block.

```suggestion
							text={ __( 'Choose Navigation Menu' ) }
```

Reply:
This is good as it adds more context. However, I think that wording will be too long for the space in the toolbar.

Can we use a `aria-label` attribute here to _enhance_ the existing wording?

Reply:
Update: I [suggested using the name of the menu as the visible text](https://github.com/WordPress/gutenberg/pull/54214#discussion_r1319709063) and then using the wording `Choose Navigation Menu` as the `aria-label`.

Reply:
Dome